### PR TITLE
MFA Support via ENV Variables

### PR DESCRIPTION
**Pull request by david-christensen**
This is a 🙋‍♂️ feature or enhancement.

- [ ] I've added tests (if it's a bug, feature or enhancement)
- [x] I've adjusted the documentation (if it's a feature or enhancement)
- [x] The test suite passes (run `bundle exec rspec` to verify this)

## Summary

Changes here add support for using MFA for `jets deploy` (and `jets delete`), via the use of environment variables. I've been wrapping my jets deploys in a secure note script and I've added documentation that exhibits that approach.

I've manually tested the changes:
- no MFA setup
- MFA with an assumed role
- MFA without an assumed role

I haven't had time to add unit tests for the changes, but wanted to get this out in case others may want to use the functionality. Others are also welcome to do so.

## Context

https://github.com/tongueroo/jets/issues/213

### CODE REVIEW

File: docs/_docs/extras/deploying-with-mfa.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: Deploying with MFA
+nav_order: 89
```
Comment (david-christensen) [inline docs/_docs/extras/deploying-with-mfa.md]: Not sure where this should go. I'm guessing I'll have to update the other `nav_order` values too. 🤷‍♂

Comment (tongueroo) [reply docs/_docs/extras/deploying-with-mfa.md]: Nope, you can leave the nav_order as-is.  Will figure out where it goes and the CI process will update it accordingly.

**Comment by tongueroo** [comment]
Thanks for this PR! Took a look. Would be nice if `jets deploy` prompted for MFA and handled the env vars transparently. Will leave PR open and around for now.

**Comment by tongueroo** [comment]
Done in #402 instead. 

Thanks for this PR though. Helped understand what needed to be done.

**Pull request closed without merging**